thirdparty/patches/rocksdb-5.14.2.patch

```diff
@@ -9,3 +9,88 @@
    std::queue<T> buffer_;
    bool eof_;
  };
+diff --git a/include/rocksdb/metadata.h b/include/rocksdb/metadata.h
+--- a/include/rocksdb/metadata.h
++++ b/include/rocksdb/metadata.h
+@@ -15,42 +15,6 @@
+ 
+ namespace rocksdb {
+ struct ColumnFamilyMetaData;
+-struct LevelMetaData;
```

Comment:
I did not find any changes. I find you only change the sequence of these code, including thrift defenitions. Anything wrong in previous code?

Reply:
By changing the declaration order, the forward declaration is removed, then T is not incomplete type when defining vector<T>.

Reply:
```cpp
struct Inner;
struct S {
  std::vector<Inner> v;
};

struct Inner {
  int a;
};
```
This is a UB example.